cookbook/ui_customization/create_a_reference_data_crud.rst

```diff
@@ -7,11 +7,57 @@ How to create the UI to manage a Reference Data
 Creating the Reference Data Management UI
 *****************************************
 
+The custom entity
+-----------------
+
+Reusing the Color entity from :doc:`/cookbook/catalog_structure/creating_a_reference_data` as example, we will first transform it into a custom entity.
+Instead of extending ``AbstractReferenceData``, it needs to extends ``AbstractCustomEntity`` (which itslef extends ``AbstractReferenceData``) as follow:
+
+.. code-block:: php
+
+    <?php
+    # /src/Acme/Bundle/AppBundle/Entity/Color.php
+
+    namespace Acme\Bundle\AppBundle\Entity;
+
+    use Pim\Bundle\CustomEntityBundle\Entity\AbstractCustomEntity;
+
+    /**
+     * Acme Color entity
+     */
+    class Color extends AbstractCustomEntity
+    {
+        ...
```

Comment:
to remove

Reply:
Mh. I wanted to show there is actual code here from the previous cookbook, not only what we see here. Maybe a comment would be better?